src/validation.h

```diff
@@ -174,8 +174,9 @@ static const int MAX_UNCONNECTING_HEADERS = 10;
 
 static const bool DEFAULT_PEERBLOOMFILTERS = true;
 
-static const uint64_t DEFAULT_GAS_LIMIT=190000;
-static const CAmount DEFAULT_GAS_PRICE=0.0000001*COIN;
+static const uint64_t DEFAULT_GAS_LIMIT_OP_CREATE=1000000;
+static const uint64_t DEFAULT_GAS_LIMIT_OP_SEND=200000;
+static const CAmount DEFAULT_GAS_PRICE=0.00000001*COIN;
```

Comment:
These could probably be moved to another file besides validation.h, since they aren't consensus parameters